PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -33,9 +34,13 @@
       </ul>
       <h4>Fixed</h4>
       <ul>
-        <li>Fixed some bugs:
+        <li>
```

Comment:
And some description here to align the format.  `Runtime information is not clear enough for Azure Web Apps`.
``` html
<li>Issue discrption <a href="links" /></li>
```

Reply:
NP, but I prefer add the issue link at the beginning.

Reply:
Like this:
![image](https://user-images.githubusercontent.com/7385667/45733740-713d2300-bc14-11e8-9aca-8ccafc69615f.png)

(There's some changed not updated.)